moveit_ros/moveit_servo/include/moveit_servo/servo_component.h

```diff
@@ -45,10 +45,10 @@
 
 namespace moveit_servo
 {
-class ServoServer : public rclcpp::Node
+class ServoComponent : public rclcpp::Node
```

Comment:
What are the implications of calling this class Component? Isn't it a node? So `ServoNode` makes most sense to me, which in turn can be a component node (exported, no class/file needed) or a node executable `ServoNodeExecutable`.

Reply:
Changes here should be reviewed along with #621 (Add standalone executable for Servo server node). In fact, maybe this PR should be closed and we just make all of the naming changes as a separate commit in that PR.

I think `ServoNodeComponent` (class name) and `servo_component_node.cpp` make good sense for this class / file.

The executable that Joe added in #621 could be called `ServoMain` or `ServoStandaloneNode`. It just instantiates this class.

Reply:
+1 to `ServoNode`. In my mind we create a node and then allow launching it in a component container.